flink-table-store-core/src/main/java/org/apache/flink/table/store/file/casting/FieldGetterCastExecutor.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.store.file.casting;
+
+import org.apache.flink.table.store.data.InternalRow;
+
+import javax.annotation.Nonnull;
+
+/** Read data from row with given pos and cast it according to specific {@link CastExecutor}. */
+public class FieldGetterCastExecutor {
```

Comment:
it provides `getFieldOrNull`, it should be a getter instead of Executor.
So rename this to `CastFieldGetter`?